glslc/src/file_compiler.h

```diff
@@ -75,6 +78,9 @@ class FileCompiler : public shaderc_util::Compiler {
   // Returns the name of the output file, given the input_filename string.
   std::string GetOutputFileName(std::string input_filename);
 
+  // Resoloves relative paths against this working directory.
```

Comment:
Consider "Relative paths of files on the command line are resolved against this directory."